tests/unit/components/search/SearchForm.spec.js

```diff
@@ -77,8 +77,13 @@ const entityApiSuggestionsResponse = {
   ]
 };
 const parsedSuggestions = {
-  'http://data.europeana.eu/concept/base/227': { en: 'Fresco' },
-  'http://data.europeana.eu/agent/base/59981': { en: 'Frank Sinatra' }
+  'Fresco': {
```

Comment:
Get rid of this block. What's here now is not what is passed in as 'suggestions' anyway.
The object passed in is still based of the entity URIs, but these are not utilized for any of the current functionality. We could potentially refactor this now, but I think the idea of linking the suggestions to entity(collection) pages is still valid so may be re-implemented later.

Also these data.europeana uris should be unique, which is why they make good keys for this array in the actual code.